src/ui/units/wizard/selectors/dialogColor.ts

```diff
@@ -55,3 +64,26 @@ export function isGradientDialog(
 
     return false;
 }
+
+const VISUALIZATION_IDS_WITHOUT_COLOR_MODE_CHANGE = new Set<string>([
+    VISUALIZATION_IDS.LINE,
+    VISUALIZATION_IDS.AREA,
+    VISUALIZATION_IDS.AREA_100P,
+]);
+
+export function isColorModeChangeAvailable(
+    props: Pick<OpenDialogColorArgs['props'], 'item'> & {
+        visualizationId?: string;
+    },
+) {
+    const {item, visualizationId} = props;
+
+    const itemIsSuitable =
+        item.type === DatasetFieldType.Dimension && item.data_type === DATASET_FIELD_TYPES.INTEGER;
```

Comment:
is the float type not considered suitable for a gradient?

Reply:
According to Pavel's plan - yes

Reply:
But I think it will be ok